docs/code-signing.rst

```diff
@@ -0,0 +1,10 @@
+Code signing policy
+===================
+
+The code used by the DebOps Project and commited in its repositories is
+expected to be signed by a valid GPG key of a DebOps Developer or DebOps
+Contributor. This should allow for secure code authentication and a valid audit
```

Comment:
I am currently working on this section. I guess you are right. It mainly affects the core devs and should be optional for now for contributes as disused with @drybjed. To your questions how to use GPG and so one, there are good answers to those questions. To the OpSec of developers you mentioned. You are right. That is a difficult topic that can not be solved by GPG/PGP alone but it is a important part of it.